at_commons/lib/src/keystore/at_key.dart

```diff
@@ -281,6 +298,18 @@ class PrivateKey extends AtKey {
   }
 }
 
+/// Represents a local key
```

Comment:
Dart docs should indicate behaviour of the local keys

Reply:
Changes to not syncing them are not here? Is it intentional?

Reply:
The documentation related to "Local keys do not get synced" are added to the "LocalKeyBuilder" class. Will update the docs to the LocalKey class also.